deploy/future

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+# Don't run this with -e, grep exits with 1 when it finds no results
+
+function check_py () {
+# This is a bit janky. If you know how to get the output
+# to go on separate lines, please tell me.
+    results=`git --no-pager grep $1|grep "\.py"|grep -v "future-ok"`
```

Comment:
I intentionally used something different from what flake8 uses because they're testing different things. I'm not too happy with the name I came up with, though--do you have a better idea? Or do you think I should just not worry about the collision?

The `--no-pager` was necessary when I added it. I can check again, since I might have changed something relevant, but I think it probably needs to be there.